packages/rum-core/src/boot/rumPublicApi.spec.ts

```diff
@@ -602,6 +603,53 @@ describe('rum public api', () => {
     })
   })
 
+  describe('addFeatureFlags', () => {
+    let addFeatureFlagsSpy: jasmine.Spy<ReturnType<StartRum>['addFeatureFlags']>
+    let displaySpy: jasmine.Spy<() => void>
+    let rumPublicApi: RumPublicApi
+    let setupBuilder: TestSetupBuilder
+
+    beforeEach(() => {
+      addFeatureFlagsSpy = jasmine.createSpy()
+      displaySpy = spyOn(display, 'error')
+      rumPublicApi = makeRumPublicApi(
+        () => ({
+          ...noopStartRum(),
+          addFeatureFlags: addFeatureFlagsSpy,
+        }),
+        noopRecorderApi
+      )
+      setupBuilder = setup()
+    })
+
+    afterEach(() => {
+      setupBuilder.cleanup()
+    })
+
+    it('should allow to add feature flags before init', () => {
```

Comment:
```suggestion
    it('should collect feature flags from before init', () => {
```

Reply:
I updated the test to be "API should not be available when ff feature_flags disabled"